src/main/java/duke/system/Parser.java

```diff
@@ -0,0 +1,172 @@
+package duke.system;
+
+import duke.system.exception.DukeException;
+import duke.task.*;
```

Comment:
I think it is better to list out the imports rather than importing by *

Reply:
Agree! :)